src/types.ts

```diff
@@ -16,9 +16,19 @@ export type RenderEnzyme<
   ...testProps: ConditionallyRequiredTestProps<Component, Props>
 ) => RenderEnzymeReturn<Component>;
 
+// Just like RenderEnzyme, but RenderRtl also allows callers to do `renderRtl.options({...})(props)`
+// In order to support both signature types, we extend the function's base type and add the options
+// attribute for those "in the know" ;)
 export type RenderRtl<
   Component extends React.ComponentType,
   Props extends Partial<FullProps<Component>>
+> = RenderRtlMethod<Component, Props> & {
```

Comment:
Is there a reason you opted to make an `&` here instead of having `RenderRtlMethod` have an `options` member?

```ts
type RenderRtlMethod<...> = {
   (...) =>...
   options(...): ...
}
```

No preference on my end, just checking that it's intentional 😎

Reply:
The only intentional part was that I didn't know there was this yet-more-elegant way to do it!